package.json

```diff
@@ -17,12 +17,12 @@
     "auth0": "^2.5.0",
     "bluebird": "^3.4.6",
     "bunyan": "~1.8.1",
-    "bunyan-sumologic": "^1.0.1",
+    "bunyan-sumologic": "avnersorek/bunyan-sumologic",
```

Comment:
I hope this gets merged in to the main repo and then I'll delete it and we can get it from NPM